src/pages/documentation/adobe-analytics/api-reference.md

```diff
@@ -15,7 +15,11 @@ Force delete, without sending to Analytics, all hits being stored or batched on
 
 Use caution when manually clearing the queue. This operation cannot be reverted.
 
-<TabsBlock orientation="horizontal" slots="heading, content" repeat="6"/>
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="7"/>
+
+Android 
+
+<Tabs query="platform=android&api=clear-queue"/>
```

Comment:
For all these tabs here, did we uncomment the content or something? (This is me not remembering what was done within the tab files).

Reply:
The Android tab didn't exist for Analytics, so I am adding them there.  I looked at the other extension, we do need Android tab for previous version right?

Reply:
Yup! That would be correct. I'm just... more surprised that Android wasn't here for some reason that's all!